ansible_collections/arista/avd/plugins/plugin_utils/utils/load_python_class.py

```diff
@@ -0,0 +1,44 @@
+import importlib
+from typing import Optional
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+
+
+def load_python_class(module_path: str, class_name: str, parent_class: Optional[type] = None) -> type:
+    """
+    Load Python Class via importlib
+
+
+    Parameters
+    ----------
+    module_path : str
+        Path to import the module from
+    class_name : str
+        Name of the class to load
+    parent_class : type
+        Class from which the imported class must inherit if present
+
+    Returns
+    -------
+    type
+        The loaded Class (and not an instance of the Class)
+
+    Raises
+    ------
+    AristaAvdError
+        If module_path or class_name are not present
+        If importlib fails to load the Class
+        If the loaded Class is not inheriting from the optional parent_class
+    """
+    if not module_path or not class_name:
+        raise AristaAvdError("Cannot load a class from a module if class_name or module_path are not given")
```

Comment:
This is not clear in the context of someone forgetting to set the module under `templates` yaml. Maybe include the information we have to guide them.

Reply:
inside the method I don't have much information actually I don't know if we are inside templates or loader but will make this better